packages/functional-tests/tests/postVerify/accountRecovery.spec.ts

```diff
@@ -3,44 +3,27 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { test, expect } from '../../lib/fixtures/standard';
-const password = 'passwordzxcv';
-let email;
 
 test.describe('severity-1 #smoke', () => {
   test.describe('post verify - account recovery', () => {
     test.beforeEach(async ({ target, pages: { login } }) => {
       // Generating and consuming recovery keys is a slow process
       test.slow();
-      email = login.createEmail();
```

Comment:
I wonder if there is a reason why we manually create a new account via the backend here.

Reply:
AFAIR, this was kind of just a short-cut way to create account when you don't want it to be pre-verified and to go through the hassle of creating an account and go through the whole steps